Better insectoid glasses (round 2)

HTMLSystem opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
This adds custom sprites for arachnids and moths for all glasses and huds. 
## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
It's so the glasses actually cover the species eyes. There are many players, myself included, that avoid wearing glasses and some huds (or have to use the eyepatches) because they look dumb on those species. This fixes that.
## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
Sprite stuff and whatnot. 
## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.
-->
![353158625-8226dbde-07bf-4c16-88e7-fda94e5c0c1c](https://github.com/user-attachments/assets/74eec0cf-becd-4327-9b8b-b349aa420151) ![353158648-2b4832d7-5157-40e3-aa11-124567cf6f49](https://github.com/user-attachments/assets/aef9c578-5a94-42c0-8631-daf85fbe241b) ![353158661-0b10124c-d04c-41de-8231-3527e9d8df3b](https://github.com/user-attachments/assets/1dc2411e-8206-4f88-a555-946625d36696)
![353158674-39825ccb-f5ce-43da-a30c-d72f9a6fe87d](https://github.com/user-attachments/assets/22167d76-f6fe-4c7b-bc7f-c76e9a95852d) ![353158683-c831ff56-55c6-41a1-aecb-289088cef589](https://github.com/user-attachments/assets/a3b00456-2cb6-4241-a25d-ce4d068a41cf) ![353158688-4ec7bc6e-ed92-4d4f-84d2-9a3811422ad3](https://github.com/user-attachments/assets/eebc58ef-0057-4c73-9a96-d49f12b57725)


## Requirements
<!-- 
Due to influx of PR's we require to ensure that PR's are following the correct guidelines.

Please take a moment to read these if its your first time.

Check the boxes below to confirm that you have in fact seen these (put an X in the brackets, like [X]):
-->
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->
N/A
**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->
Nuh uh
<!--
Make sure to take this Changelog template out of the comment block in order for it to show up. Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog.
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

HTMLSystem committed:
Sprites and meta files, part 1

Adds the glasses and meta files for glasses and huds that don't require admins to spawn in.
For some reason the arachnid and moth had it's - type:Inventory section down at the bottom, so I removed that and added whatever was there to the original spot I had them in the previous PR.

HTMLSystem committed:
Sprites and meta files, part 2 admeme edition

The other part of the sprites and meta files, consisting of huds only admins can spawn in and the ninja visor.

HTMLSystem commented:
This is ready to be merged whenever.

HTMLSystem commented:
My habit of mispelling words strikes again.
I'll fix those later. In the meantime you could dm me on discord (or I could dm you, whichever happens first) on PRing this upstream.

HTMLSystem committed:
Typo fix

Fixes all typos in the meta.json files, hopefully.

HTMLSystem committed:
Merge branch 'master' into Better-insectoid-glasses-(round-2)

Signed-off-by: HTML/Crystal <152909599+HTMLSystem@users.noreply.github.com>

HTMLSystem commented:
So, the PR upstream has been in review for about a month and they seem a bit dead set on me using displacement maps for some of the glasses (which I'm not very willing to do). Would there be any problems if this is merged early here instead of waiting on upstream?
Edit: I just realized I NEVER put the command glasses or admin hud glasses in here. I'll fix that really quick.

HTMLSystem committed:
Added command  glasses and hud

HTMLSystem commented:
The test fail seems unrelated? My dev environment boots up fine without any issues or warnings, and the yaml looks completely fine. None of the warnings look related either. Though maybe I'm overlooking something...

deltanedas commented on Resources/Prototypes/Entities/Mobs/Species/arachnid.yml:
```diff
@@ -30,6 +30,9 @@
         amount: 5
+  - type: Inventory
+    templateId: arachnid
+    speciesId: arachnid # DeltaV - allow arachnid clothing sprite variants
```
add the speciesId bit to the existing Inventory component at the bottom (dont remove it just to place it here)

deltanedas commented on Resources/Prototypes/Entities/Mobs/Species/arachnid.yml:
```diff
@@ -125,6 +126,8 @@
   - type: HumanoidAppearance
     species: Arachnid
+  - type: Inventory
+    speciesId: arachnid #DeltaV - Allows the "dummy" in th elobby menu to wear the arachnid sprites
```
th elobby :trollface:

deltanedas commented on Resources/Prototypes/Entities/Mobs/Species/moth.yml:
```diff
@@ -15,6 +15,14 @@
     sprite: Mobs/Species/Moth/parts.rsi
     state: full
+  - type: Inventory
+    speciesId: moth # Deltav - allows moth clothing sprite variants
```
same, dont move it for no reason

deltanedas reviewed (changes requested):
ideally there would be a verb on clothing or a character preference to disable using species specific sprites

HTMLSystem committed:
Deltanedas requested changed

Fixed the command glasses sprite and moved the species ID to the bottom of the prototype file like I was asked to.

Commit:
merge master and fix some comments

HTMLSystem commented:
Closing because it was merged upstream!!
